dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -109,4 +84,122 @@ def report_abuse_form
       age: current_user&.age,
     }
   end
+
+  # API routes, moved from legacy channels_api.rb and files_api.rb.
+
+  # GET /v3/channels/:channel_id/abuse
+  # Get an abuse score.
+  def show_abuse
+    begin
+      value = Projects.get_abuse(params[:channel_id])
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      raise ActionController::BadRequest.new, "Bad channel_id"
+    end
+    render json: {abuse_score: value}
+  end
+
+  # DELETE /v3/channels/:channel_id/abuse
+  # Clear an abuse score. Requires project_validator permission
+  def destroy_abuse
```

Comment:
nit: this might be just me, but "destroy" makes it seem we're deleting something rather than updating it. `reset_abuse` makes more sense to me here.